include/pokemon.h

```diff
@@ -385,6 +385,7 @@ struct BattleMove
     u32 copycatBanned:1;
     u32 sleepTalkBanned:1;
     u32 instructBanned:1;
+    u32 powerfullMoveEffect:1;
```

Comment:
Minor nitpick: should be `powerfulMoveEffect` with one 'l'.

But I think it would probably be helpful to give this a different name that makes it more clear what it's for; i.e. it's only used by the AI, and it's kinda a `banned` thing in that it excludes the move from certain calculations (I think?).